### ISSUE TITLE
"麻将牌 JSON"的表达可能有失妥当

### ISSUE THREAD

**Issue by Water-Buckets** [Fallen-Breath/MinecraftDataAPI#19]
"麻将牌 JSON"或"Minecraft style json"似乎并不是一个合适的说法. 更加精确的来说, 在Minecraft 中它被称为"SNBT格式".
使用"麻将牌 JSON"这样带有戏谑意味的词的确是可以理解的, 这确实体现了幽默感. 但是 MCDR 作为一个被广泛接受的服务器工具, 它的插件系统中包括了许多插件, 这其中有相当一部分插件都使用了您的MinecraftDataAPI. 考虑到其被广泛的使用, 这样的名字可能带来困惑. 考虑到大部分插件作者的技术以及认知水平, "SNBT"可能是一个更加合适的选择. 
当然, 仅仅因为一个有趣的名字而改动 API 是不合适的, 相比更加有用的工作, 改动这个名字也没有那么重要. 以上内容请您参考.
最后, 也非常感谢您为 MCDR 的生态所做的贡献.

**Comment by Fallen-Breath** [comment]
> 考虑到其被广泛的使用

这里出现了偷换概念的行为。前文是以用户的角度来描述 MinecraftDataAPI 的使用范围之广，而本 issue 所描述的用词是位于面向插件开发者的 API 文档中，是面向开发者的。将面向用户的“广泛”定语应用到开发者的文档上，是存在概念偷换的。广泛使用本插件的普通用户并不能感知到开发者文档中的这一用词

----

回到这一 issue，若认为“麻将牌 json”这一表述存在含糊不清的，我之后可以将相关用词改造成指向 mc wiki 的超链接，这样也不会出现表述不清的现象

**Comment by Water-Buckets** [comment]
> > 考虑到其被广泛的使用
> 
> 这里出现了偷换概念的行为。前文是以用户的角度来描述 MinecraftDataAPI 的使用范围之广，而本 issue 所描述的用词是位于面向插件开发者的 API 文档中，是面向开发者的。将面向用户的“广泛”定语应用到开发者的文档上，是存在概念偷换的。广泛使用本插件的普通用户并不能感知到开发者文档中的这一用词

是的, 我的本意是有许多插件都将 MinecraftDataAPI 作为依赖, 是插件开发者使用的较多. 确实我的表述存在问题.

-----

> 回到这一 issue，若认为“麻将牌 json”这一表述存在含糊不清的，我之后可以将相关用词改造成指向 mc wiki 的超链接，这样也不会出现表述不清的现象

如果能修改原有的函数名是最好不过的了. 但是作为一个被开发者广泛使用的插件, 仅仅修改文档也是合理的.

附: 
[SNBT格式](https://zh.minecraft.wiki/w/NBT%E6%A0%BC%E5%BC%8F#SNBT%E6%A0%BC%E5%BC%8F)
[教程:SNBT](https://zh.minecraft.wiki/w/Tutorial:SNBT)
[教程:NBT命令标签](https://zh.minecraft.wiki/w/Tutorial:NBT%E5%91%BD%E4%BB%A4%E6%A0%87%E7%AD%BE)
